ajax/libs/sinon.js/package.json

```diff
@@ -16,6 +16,15 @@
     "type": "git",
     "url": "https://github.com/sinonjs/sinon"
   },
+  "npmName": "sinon",
+  "npmFileMap": [
+    {
+      "basePath": "pkg",
+      "files": [
+        "**/+(sinon.|sinon-no-sourcemaps.)*"
```

Comment:
I prefer to use a rule to grab all files without those filename containing, so that it can fit also the old versions, what do you think?